completion/available/defaults.completion.bash

```diff
@@ -56,7 +56,7 @@ _defaults()
 
 	# Both a domain and command have been specified
 
-	if [[ ${COMP_WORDS[1]} == [${cmds// /|}] ]]; then
+	if [[ ${COMP_WORDS[1]} =~ [${cmds// /|}] ]]; then
```

Comment:
How did the `=~` sneak into here?

We have at least 2 other uses of the `==` version in this, and I _thought_ we discussed previously that `$cmds` is not a regex?  I'll have to look for that discussion.

Reply:
woah, I completely missed it!
That's totally my bad... 

We should opt for the solution with `_bash-it-array-contains-element` instead.
I have also seen other problems with this completion...
I will try to create a PR for it

Reply:
It didn't sneak in. Using `==` makes the comparison case-sensitive, which was the whole point of #188. I've opened #1922 to restore it.